src/group/tests/kat_transcripts.rs

```diff
@@ -36,6 +36,7 @@ pub struct TranscriptTestVector {
     tree_hash_before: String,
     confirmed_transcript_hash_before: String,
     interim_transcript_hash_before: String,
+    credential: String,
```

Comment:
I assume this is included so that the signature on the commit can be checked as well? Then we should definitely file an issue to re-include the mlspp test vectors and probably also to change the test vector definitions.